### PR TITLE
Fix deepsource issues

### PR DESCRIPTION
**Pull request by dITySoftware**

### CODE REVIEW

File: source/root/etc/aminer/conf-available/generic/ApacheAccessModel.py [hunk 1]
```diff
@@ -8,6 +8,7 @@
 from aminer.parsing import AnyByteDataModelElement
 from aminer.parsing import FixedWordlistDataModelElement
 
+# skipcq: PYL-W0613
```
Comment (landauermax) [inline source/root/etc/aminer/conf-available/generic/ApacheAccessModel.py:11]: Der Parameter müsste new_time_model heißen

File: source/root/etc/aminer/conf-available/generic/RsyslogParsingModel.py [hunk 2]
```diff
@@ -6,6 +6,7 @@
 from aminer.parsing import FixedDataModelElement
 from aminer.parsing import SequenceModelElement
 
+# skipcq: PYL-W0613
```
Comment (landauermax) [inline source/root/etc/aminer/conf-available/generic/RsyslogParsingModel.py:9]: Parameter löschen

File: source/root/usr/lib/logdata-anomaly-miner/aminer/analysis/HistogramAnalysis.py [hunk 3]
```diff
@@ -56,13 +56,14 @@
 try:
   from scipy import stats
   binomial_test = stats.binom_test
+# skipcq: FLK-E722
```
Comment (landauermax) [inline source/root/usr/lib/logdata-anomaly-miner/aminer/analysis/HistogramAnalysis.py:59]: Ok

File: source/root/usr/lib/logdata-anomaly-miner/aminer/analysis/HistogramAnalysis.py [hunk 4]
```diff
@@ -88,6 +89,7 @@ def get_bin(self, value):
     at index 1."""
     raise Exception(self.not_implemented)
 
+  # skipcq: PYL-W0613
```
Comment (landauermax) [inline source/root/usr/lib/logdata-anomaly-miner/aminer/analysis/HistogramAnalysis.py:92]: Maybe add in comment that this is just used as an interface method

File: source/root/usr/lib/logdata-anomaly-miner/aminer/parsing/DateTimeModelElement.py [hunk 5]
```diff
@@ -18,6 +18,7 @@ class DateTimeModelElement(ModelElementInterface):
   in libc strptime, no support to determine the length of the
   parsed string."""
 
+  # skipcq: PYL-W0613
```
Comment (landauermax) [inline source/root/usr/lib/logdata-anomaly-miner/aminer/parsing/DateTimeModelElement.py:21]: Ok, it is later written in comment that this part is not yet implemented

File: source/root/usr/lib/logdata-anomaly-miner/aminer/analysis/HistogramAnalysis.py (hunk 4 above)
Comment (dITySoftware) [reply source/root/usr/lib/logdata-anomaly-miner/aminer/analysis/HistogramAnalysis.py:92]: Done

File: source/root/etc/aminer/conf-available/generic/RsyslogParsingModel.py (hunk 2 above)
Comment (dITySoftware) [reply source/root/etc/aminer/conf-available/generic/RsyslogParsingModel.py]: Done.

File: source/root/etc/aminer/conf-available/generic/ApacheAccessModel.py (hunk 1 above)
Comment (dITySoftware) [reply source/root/etc/aminer/conf-available/generic/ApacheAccessModel.py:12]: Done.

**Pull request merged**